package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ldclient-node",
-  "version": "4.0.3",
+  "version": "4.0.4",
```

Comment:
Isn't the next version 4.0.5?

Reply:
I never manually change these numbers -- i had to run `npm install` locally and that was bumped. Not sure if we're supposed to manually update this number as well.

I'll check